_posts/2024-03-15-customer-experience-outcomes.md

```diff
@@ -0,0 +1,64 @@
+
+When navigating software, users utilize two types of memory: episodic and semantic. Episodic memory is grounded in the sequence of time, while semantic memory is conceptual. We, as humans, are capable of experiencing our online environment in both ways and often switch between the two types of memory as we navigate software. When functionality is enhanced incrementally, however, navigation that fails to account for the logical buildup of user tasks in a software workflow can quickly degrade the experience. Such product experiences end up ill designed and requiring ongoing customer support to help coach, train, and assist the user.
+
+## Is there a way to measure user experience?
+The [Customer Outcome Framework](https://productmanagementuniversity.com/launches-customer-outcome-framework/) employs a top-down view of the customer experience. Customer Experience Outcomes (CXOs) [define critical tasks that a product must perform and how well those tasks must be performed](https://lithespeed.com/drive-input-customer-experience-outcomes/#:~:text=What%20is%20a%20Customer%20Experience%20Outcome%20or,a%20plan%20for%20success:%20%E2%80%9CIf%20we%20do). A comprehensive set of CXO measures may include [quality and loyalty measures](https://www.forrester.com/research/cx-index/). Additional outcome measures may include [satisfaction and frustration scores](https://www.dynatrace.com/news/blog/user-experience-score-the-one-metric-to-rule-them-all/), [UMUX](https://testscience.org/measuring-usability/#UMUX)[measures](https://testscience.org/measuring-usability/#UMUX), [UMUX-lite measures](https://testscience.org/measuring-usability/#UMUX-LITE), or [SUS scores](https://testscience.org/measuring-usability/#SUS).
+
+## OpenSearch experience outcomes
+To measure OpenSearch CXOs, we first defined [distinct users](https://opensearch.org/blog/q1-survey-results/) based on roles and domain expertise. Once we validated these, we then parsed out pivotal nodes of each user’s experience as user flows. We intentionally limited the pivotal experience nodes to 3–6 in number so that we could manage the length of the survey. For each node, we asked users to provide ratings for four measures representing their perception of the experience. We adopted UMUX scale measures to quantify satisfaction, frustration, usefulness, and usability. After each set of ratings, we invited our users to tell us how we might improve. The survey was hosted for two quarters (Q3–Q4 2023), and we obtained 130 responses in total from OpenSearch Dashboards users.
```

Comment:
```suggestion
To measure OpenSearch CXOs, we first defined [distinct users](https://opensearch.org/blog/q1-survey-results/) based on roles and domain expertise. Once we validated them, we parsed pivotal nodes of each user’s experience as user flows. We intentionally limited the pivotal experience nodes to 3–6 in number so that we could manage the length of the survey. For each node, we asked users to provide ratings for four measures representing their perception of the experience. 

We adopted UMUX scale measures to quantify satisfaction, frustration, usefulness, and usability. After each set of ratings, we invited the users to tell us how we might improve. The survey was hosted for two quarters (Q3 and Q4 2023). We received 130 responses from OpenSearch Dashboards users. Data is shown in the following images.
```